src/app/common-components/bread-crumbs/bread-crumbs.component.spec.ts

```diff
@@ -1,25 +1,30 @@
-import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
 
-import { BreadCrumbsComponent } from './bread-crumbs.component';
+import { BreadCrumbsComponent } from "./bread-crumbs.component";
 
-describe('BreadCrumbsComponent', () => {
-  let component: BreadCrumbsComponent;
-  let fixture: ComponentFixture<BreadCrumbsComponent>;
+describe("BreadCrumbsComponent", () => {
+    let component: BreadCrumbsComponent;
+    let fixture: ComponentFixture<BreadCrumbsComponent>;
 
-  beforeEach(async(() => {
-    TestBed.configureTestingModule({
-      declarations: [ BreadCrumbsComponent ]
-    })
-    .compileComponents();
-  }));
+    beforeEach(async(() => {
+        TestBed.configureTestingModule({
+            declarations: [BreadCrumbsComponent]
+        }).compileComponents();
+    }));
 
-  beforeEach(() => {
-    fixture = TestBed.createComponent(BreadCrumbsComponent);
-    component = fixture.componentInstance;
-    fixture.detectChanges();
-  });
+    beforeEach(() => {
+        fixture = TestBed.createComponent(BreadCrumbsComponent);
+        component = fixture.componentInstance;
+        fixture.detectChanges();
+    });
 
-  it('should create', () => {
-    expect(component).toBeTruthy();
-  });
+    it("should create", () => {
+        expect(component).toBeTruthy();
+    });
+
+    it("should generate complete path of the module", () => {
+        component.path = ["A", "B"];
+        component.ngOnChanges();
```

Comment:
it is not a good idea to call life cycle methods, usually you can call fixture.detectChanges();